src/shipit_signoff/default.nix

```diff
@@ -21,7 +21,7 @@ let
       export LOCALE_ARCHIVE=${glibcLocales}/lib/locale/locale-archive
       export APP_TESTING=${name}
 
-      flake8 --exclude=nix_run_setup.py,migrations/,build/
```

Comment:
You can remove the whole ``checkPhase`` now that tests are working again.

If you want to play a little with nix you can also extend ``mkBackend`` (and ``mkPython``) function to enable extra configuration options and lets you turn on/off flake8 and pytest. eg.

```
self = mkBackend {
  inherit python name dirname;
  version = fileContents ./VERSION;
  src = filterSource ./. { inherit name; };
  doCheckStyle = true;
  doCheckTests = false;
}
```

Above would only run ``flake8`` linter and skip ``pytest tests/``. If you have some Nix syntax questions you can ping me.